modules/using-a-drift-metric-in-a-credit-card-scenario.adoc

```diff
@@ -0,0 +1,204 @@
++
+[source, bash]
+----
+TRUSTY_ROUTE=https://$(oc get route/trustyai-service --template={{.spec.host}})
+----
+
+. Download the training data payload (472 KB):
++
+[source, bash]
+----
+wget https://github.com/trustyai-explainability/odh-trustyai-demos/blob/72f748da9410f92a60bea73ce5e3f47c10ad1cea/3-DataDrift/kserve-demo/data/training_data.json -O training_data.json 
+----
+
+. Label your model training data. This data has four main fields. Two of these require a label(`model_name` and `data_tag`) because these fields will be directly referenced in the Metrics dashboard's output later in the scenario. We recommend that you also label the other two main fields.
++
+The four fields are:
+.. `model_name`: The name of the model that correlates to this data. The name should match that of the model provided in the model yaml, which is `gaussian-credit-model`.
+.. `data_tag`: A string tag to reference this particular set of data. Use the string "TRAINING"
+.. `request`: This is a KServe inference request, as if you were sending this data directly to the model server's `/infer` endpoint.
+.. `response`: The KServe inference response that is returned from sending the above request to the model.
+
+. Upload the model training data to the TrustyAI endpoint:
++
+[source, bash]
+----
+curl -sk -H "Authorization: Bearer ${TOKEN}" $TRUSTY_ROUTE/data/upload  \
+ --header 'Content-Type: application/json' \  
+ -d @training_data.json 
+----
++
+The message "1000 datapoints successfully added to gaussian-credit-model data" appears.
+
+. (Recommended) Label your model input and output fields with the actual column names of the data in your KServe payloads. Send a JSON payload containing a simple set of `original-name` : `new-name pairs`, assigning new meaningful names to the input and output features of your model. The message "Feature and output name mapping successfully applied" appears if the request is successful:
++
+[source, bash]
+----
+curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST --location $TRUSTY_ROUTE/info/names \
+  -H "Content-Type: application/json"   \
+  -d "{
+    \"modelId\": \"gaussian-credit-model\",
+    \"inputMapping\":
+      {
+        \"credit_inputs-0\": \"Age\",
+        \"credit_inputs-1\": \"Credit Score\",
+        \"credit_inputs-2\": \"Years of Education\",
+        \"credit_inputs-3\": \"Years of Employment\"
+      },
+    \"outputMapping\": {
+      \"predict-0\": \"Acceptance Probability\"
+    }
+  }"
+----
++ 
+[TIP]
+====
+Define name mappings in TrustyAI to assign memorable names to input or output names. These names can then be used in subsequent requests to the TrustyAI service.
+====
+
+. Verify that TrustyAI has received the data by querying the `/info` endpoint:
++
+[source, bash]
+----
+curl -H "Authorization: Bearer ${TOKEN}" $TRUSTY_ROUTE/info | jq '.["gaussian-credit-model"].data.inputSchema'
+----
+
+. The following output appears as a json file confirming that TrustyAI has successfully received the data:
```

Comment:
Possible to add an action verb here to maintain an active tone and overall consistency? Also, is this output an outcome of the verification step above? If so, it should probably be added under it? 

Should **json** be capitalized?